apps/mobile/src/hooks/use-account-assets.ts

```diff
@@ -0,0 +1,18 @@
+import { useRunesAccountBalance } from '@/queries/balance/runes-balance.query';
+import { useSip10AccountBalance } from '@/queries/balance/sip10-balance.query';
+
+export function useAccountAssets(
+  fingerprint: string,
+  accountIndex: number,
+  options?: {
+    returnAllAssets?: boolean;
+  }
+) {
+  const sip10Data = useSip10AccountBalance(fingerprint, accountIndex, options);
+  const runesData = useRunesAccountBalance(fingerprint, accountIndex, options);
+  return {
```

Comment:
Maybe you can add `isLoading` in this response also? You could even wrap it in a `toFetchState` and also pass on any errors